bin/build-fonts.js

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import SYSTEM_FONTS from './systemFonts';
+
+const { readFileSync, writeFileSync, existsSync } = require('fs');
+const process = require('process');
+
+const PLUGIN_DIR = process.cwd();
+const FONTS_FILE = PLUGIN_DIR + '/includes/data/fonts.json';
```

Comment:
Just curious. Is this fonts json static or dynamically updated? If it's kind of static data and we are using some process to reduce some memory issue, can we have font face also for that file? i.e.
font-face: ```@font-face {
  font-family: 'Alata::MENU';
  font-style: normal;
  font-weight: 400;
  src: local('Alata Regular'), local('Alata-Regular'), url(https://fonts.gstatic.com/s/alata/v1/PbytFmztEwbIoce7zqYzQ1OW.woff2) format('woff2');
}``` maybe this whole one could be big. But just `src`?

Reply:
It's a static file, regularly updated. This static list is used to fetch the actual Google Fonts CSS in the browser when needed.

I wouldn't want to include actual CSS `@font-face` declarations here. Google Fonts serves different CSS depending on your browser's capabilities.